task14/index.js

```diff
@@ -0,0 +1,109 @@
+const parent = document.getElementById("js-parent"); 
+const modalPlace = document.getElementById('modalPlace');
+
+const appendModalForParent = () => {
+    const modalText = document.createElement('p');
+    const getNumber = document.createElement('input')
+    const closeBtn = document.createElement('input');
+    modalPlace.className = "modal";
+    modalPlace.style.display = "block";
+    modalText.textContent = "数字を入力して値を取得します。";
+    getNumber.id = "js-number";
+    getNumber.className = "number";
+    getNumber.type = "number";
+    closeBtn.id = "js-closeBtn";
+    closeBtn.className = "close-button";
+    closeBtn.type = "button";
+    closeBtn.value = "click";
+
+    modalPlace.appendChild(modalText);
+    modalPlace.appendChild(getNumber);
+    modalPlace.appendChild(closeBtn);
+};
+
+const appendButtonForParent = () => {
+    const button = document.createElement('input');
+    button.id = "js-openBtn";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+appendButtonForParent();
+
+const requestNumber = () => {
+    const getValue = document.getElementById("js-number");
+    if(getValue.value === ""){
+        alert("数字を入力してください");
+        return false;
```

Comment:
IMO
The boolean value returned by return here is not used anywhere, so it seems unnecessary.
What do you think?
(Also the return on line 42)

Reply:
So if you don't use it, you don't have to write it down!
I'll be careful not to write anything useless.
9974072